### PR TITLE
Dataset issues

### PR DESCRIPTION
**Pull request by xavijam**
- Bug reported here -> https://github.com/CartoDB/cartodb/issues/11855.
- Pagination position in Safari browser.
- Renaming a table breaks edition until reloaded (https://github.com/CartoDB/cartodb/issues/11969).

We should test that all those things are tested, just follow the issues.

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/components/table/table-view.js [hunk 1]
```diff
@@ -157,6 +157,10 @@ module.exports = CoreView.extend({
     });
   },
 
+  getColumnsCollection: function () {
```
Comment (matallo) [inline lib/assets/core/javascripts/cartodb3/components/table/table-view.js]: being ultra picky here, shouldn't we have a little spec for this? even if we pass the `columnsCollection` as an option 🙏

Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/table/table-view.js]: _Bueeeeno_

**Comment by xavijam** [comment]
Moving it to acceptance, ok @matallo ?

**Comment by matallo** [comment]
acceptance straight-away

**Comment by javierarce** [comment]
@xavijam what does "Pagination position in Safari browser." mean?

All the other points in the description were successfully fixed and are accepted.

**Comment by xavijam** [comment]
There was a problem reported in twitter by a user where the pagination was placed over the create map button in the dataset view.

**Comment by javierarce** [comment]
I see, thanks. That seems to work alright too.

**Comment by xavijam** [comment]
\o/

**Comment by xavijam** [comment]
Frontend tests passed with Travis, merging after solved a conflict with master \o/.

**Pull request merged**